.github/ISSUE_TEMPLATE/issue.yml

```diff
@@ -0,0 +1,68 @@
+name: Generic issue
+description: Report an issue, a suggestion, SQL only fix.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this form! Please make sure to fill all fields, including the Title above.
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: |
+        Description of the problem or issue here.
+        If this is a crash, post the crashlog (upload to https://gist.github.com/).
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected behaviour
+      description: |
+        Tell us what should happen instead.
+    validations:
+      required: true
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: Steps to reproduce the problem
+      description: |
+        1. Step 1 include entries of affected creatures / items / quests with a link to the relevant wowhead or https://tcubuntu.northeurope.cloudapp.azure.com/aowow/ page.  
+        2. Step 2
+        3. Step 3
+    validations:
+      required: true
+  - type: dropdown
+    id: branch
+    attributes:
+      label: Branch
+      options:
+        - 3.3.5
+        - master
+    validations:
+      required: true
+  - type: textarea
+    id: commit
+    attributes:
+      label: TC rev. hash/commit
+      description: |
+        Copy the result of server debug command (if you need to run it from client get prat addon)
+    validations:
+      required: true
+  - type: textarea
```

Comment:
Would a single-line `input` field be enough? Just to reduce the vertical extent a bit.

Reply:
That could be an idea.

The issue template actually suggests to "Copy the result of server debug command (if you need to run it from client get prat addon)", this is the result of that command:
>TrinityCore rev. c8ca48823d45 2021-06-26 14:21:18 +0200 (3.3.5 branch) (Unix, RelWithDebInfo, Static)
Using SSL version: OpenSSL 1.1.1j  16 Feb 2021 (library: OpenSSL 1.1.1j  16 Feb 2021)
Using Boost version: 1.74.0
Using MySQL version: 80025
Using CMake version: 3.18.4
Compiled on: Linux 5.11.0-1006-azure
Automatic database updates are enabled for the following databases: Auth, Characters, World
Worldserver listening connections on port 8085
Realmlist (Realm Id: 1) configured in port 8085
VMAPs status: Enabled. LineOfSight: 1, getHeight: 1, indoorCheck: 1
MMAPs status: Enabled
maps directory located in /trinity/wowdata/maps. Total size: 291014951 bytes
vmaps directory located in /trinity/wowdata/vmaps. Total size: 657797442 bytes

If we want to allow pasting this, then we need a multi-row field.

Reply:
I suppose there was a misunderstanding. My comment was meant for the `os` field.
I agree that we need multiline for the `server debug` output.

Reply:
@Carbenium I picked the field above instead of the field below. OS single line sounds good